android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1123,7 +1140,6 @@ public void onCreate() {
         if (!initCheck()) { // Run checks on process and permissions
             deployNextRouterService();
             closeSelf();
-            return;
```

Comment:
Sorry for the confusion, I need you to remove code starting at 1140 to 1160 as it will get run in onStartCommand now.

Code that still needs removed in this method:

```

        if (!initCheck()) { // Run checks on process and permissions
             deployNextRouterService();
             closeSelf();
         }
         initPassed = true;

        synchronized (REGISTERED_APPS_LOCK) {
            registeredApps = new HashMap<String, RegisteredApp>();
        }
        closing = false;
        synchronized (SESSION_LOCK) {
            this.bluetoothSessionMap = new SparseArray<String>();
            this.sessionHashIdMap = new SparseIntArray();
            this.cleanedSessionMap = new SparseIntArray();
        }
        packetExecutor = Executors.newSingleThreadExecutor();
        startUpSequence();

```